src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -57,41 +51,61 @@ internal class ServicePageGenerator
                     servicePagePath = FilePath.Generated(new PathString($"./{baseDir}/{directoryName}/{filename}.yml"));
                 }
 
+                if (string.IsNullOrEmpty(node.Href.Value))
```

Comment:
`Uid` is not needed for service page, why do we need `Href`?

Reply:
This is for overview page. Although the service pages themselves don't need href of themselves, the "Overview" item should be added to ReferenceToc to get correct link to Overview page. And the link of "Overview" item is the same as the parent item, so the correct href should be added.